libsql-server/src/lib.rs

```diff
@@ -28,6 +28,7 @@ use auth::Auth;
 use config::{
     AdminApiConfig, DbConfig, HeartbeatConfig, RpcClientConfig, RpcServerConfig, UserApiConfig,
 };
+use futures::future::{pending, ready};
```

Comment:
btw these exist now in std https://doc.rust-lang.org/stable/std/future/index.html

Reply:
ah right, for some reason RA imports those by default